### PR TITLE
add SDP logo image

### PR DESCRIPTION
**Pull request by PTLux1517**
Placed below the metadata output area.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -174,7 +174,8 @@ export default function App() {
 		<div id={"output-area"}>
 			<SelectionInfoView markerData={markerFileData} selectedMarkers={selectedMarkers} frame={frame} />
 		</div>
-		{/* ---------------------------------------------- Grid Row 3 ---------------------------------------------- */}
+		<img id={"sdp-logo"} src={"https://drive.google.com/uc?export=view&id=10VJZEoeTEN7JKled94_VoUMK4c_PEfZc"} alt={"senior design project logo"} />
```
Comment (vpzomtrrfrt) [inline src/App.tsx:177]: I don't think we should be loading this from Google

Comment (PTLux1517) [reply src/App.tsx]: Updated. There were a variety of approaches for importing local images, but I'm not sure what is idiomatic for Typescript. The solution I settled on seems to work fine, hopefully it's okay.

File: webpack.config.js [hunk 2]
```diff
@@ -26,7 +26,11 @@ module.exports = {
                     "css-loader",
                     "sass-loader"
                 ]
-            }
+            },
+			{
+				test: /\.(png|jpe?g)$/,
+				loader: "url-loader"
```
Comment (vpzomtrrfrt) [inline webpack.config.js:32]: url-loader is deprecated since webpack 5, see https://webpack.js.org/guides/asset-modules/

Comment (PTLux1517) [reply webpack.config.js:32]: I've been trying to get this to work for ~45 minutes, but can't get rid of the following error:
`TS2307: Cannot find module '../assets/images/sdp-logo-3.png' or its corresponding type declarations.`

Here's the updated webpack.config.js
```js
const path = require('path');
const HtmlWebpackPlugin = require("html-webpack-plugin");

module.exports = {
	entry: './src/index.tsx',
	output: {
		filename: "[contenthash].js",
		path: path.resolve(__dirname,'dist'),
		clean: true,
	},
	resolve: {
		extensions: [".js", ".ts", ".jsx", ".tsx"]
	},
	plugins: [
		new HtmlWebpackPlugin({
			title: "Movilo",
		}),
	],
	module: {
		rules: [
			{
				test: /\.tsx?$/,
				loader: "ts-loader",
			},
			{
				test: /\.s[ac]ss$/i,
				use: [
					"style-loader",
					"css-loader",
					"sass-loader"
				 ]
			},
			{
				test: /\.png$/,
				type: 'asset/resource'
			}
		]
	}
};
```
and the problematic import in App.tsx
```typescript
import sdpLogo from "../assets/images/sdp-logo-3.png";
```

Comment (vpzomtrrfrt) [reply webpack.config.js:32]: ah right, you can set up a type definition file to deal with that: https://webpack.js.org/guides/typescript/#importing-other-assets

Comment (PTLux1517) [reply webpack.config.js:32]: Spent another ~45 mins or so. Now the TS compiler isn't complaining and my console shows that the image is being emitted, but on run dev the image isn't being displayed on the page. I created a debug branch with the current state of my changes if you want to have a look: https://github.com/cs481-ekh/f22-viz-kids/tree/debug-image. I'm not sure what else I'm missing here.

Comment (PTLux1517) [reply webpack.config.js:32]: Now I'm even more confused why using `require()` works and this doesn't.

Comment (vpzomtrrfrt) [reply webpack.config.js:32]: this seems to make it work:

```patch
diff --git a/custom.d.ts b/custom.d.ts
index d8f8deb..537e720 100644
--- a/custom.d.ts
+++ b/custom.d.ts
@@ -1,5 +1,5 @@

 declare module '*.png' {
     const content: string;
-    export default content;
+    export = content;
 }
diff --git a/src/App.tsx b/src/App.tsx
index a785b12..1065ba8 100644
--- a/src/App.tsx
+++ b/src/App.tsx
@@ -9,7 +9,7 @@ import RenderView from "./RenderView";
 import ErrorPopup from "./ErrorPopup";
 import SelectionInfoView from "./SelectionInfoView";
 import useStateRef from "./useStateRef";
-import sdpLogo from "./assets/images/sdp-logo-3.png";
+import * as sdpLogo from "./assets/images/sdp-logo-3.png";

 import "./App.scss";
```

Comment (PTLux1517) [reply webpack.config.js]: Yep that fixed it. Changes pushed

**Review by javitrejo14 — APPROVED** [review]

**Review by ConnorJacksonBSU — APPROVED** [review]

**Pull request merged**